docs/labs/flockercontainers.rst

```diff
@@ -0,0 +1,96 @@
+.. _flocker-containers:
+
+=============================
+Running Flocker in Containers
+=============================
+
+The :ref:`Flocker Installer <labs-installer>` allows you install Flocker in a variety of environments.
+For some of these environments, such as CoreOS, you can actually install Flocker inside a container.
+
+.. note::
+   Installing Flocker in containers is useful for environments such as CoreOS, RancherOS, or Amazon ECS, but please note that doing so is currently experimental.
```

Comment:
:question: Have we tested on RancherOS and Amazon ECS? Do those operating systems all have the ext4 kernel driver? Does Amazon ECS allow you to run privileged containers? And does it allow you to create a `/flocker` mount point on the host FS?
I'd suggest removing Rancher and Amazon ECS from these docs until we've tested them.